package.json

```diff
@@ -5,13 +5,13 @@
   "private": true,
   "scripts": {
     "lint": "standard",
-    "postinstall": "npm prune",
+    "postinstall": "npm run build; npm prune",
     "prestart": "npm run start:mongo",
-    "start": "nodemon -e js,jsx --exec \"nf start\"; npm stop",
+    "start": "npm run build; nodemon -e js,jsx --exec \"nf start\"; npm stop",
```

Comment:
Should probably change the `--exec` to `"npm run build && nf start"`? Otherwise nodemon restarting won't update the build.